arrows/gdal/CMakeLists.txt

```diff
@@ -0,0 +1,37 @@
+# Build / Install plugin containing GDAL algorithm implementations + support
+# structures
+
+set( gdal_headers_public
+  image_io.h
+  )
+
+kwiver_install_headers(
+  SUBDIR     arrows/gdal
+  ${gdal_headers_public}
+  )
+
+kwiver_install_headers(
+  ${CMAKE_CURRENT_BINARY_DIR}/kwiver_algo_gdal_export.h
+  NOPATH   SUBDIR     arrows/gdal
+  )
+
+set( gdal_sources
+  image_io.cxx
+  )
+
+kwiver_add_library( kwiver_algo_gdal
+  ${gdal_headers_public}
+  ${gdal_sources}
+  )
+target_link_libraries( kwiver_algo_gdal
+  PUBLIC               kwiver_algo_core
```

Comment:
It's hard for me to tell for sure, but it looks like the GDAL headers are only included from your cxx code. If that's the case, you should be able to make this link PRIVATE which is preferable to PUBLIC.

Reply:
Fixed